From: Ruben Undheim <ruben.undheim@gmail.com>
Date: Wed, 11 Jul 2018 15:47:59 +0200
Subject: Fixes the install command for the python part

---
 python/pcaptools/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/pcaptools/CMakeLists.txt b/python/pcaptools/CMakeLists.txt
index 78cfec9..3758d69 100644
--- a/python/pcaptools/CMakeLists.txt
+++ b/python/pcaptools/CMakeLists.txt
@@ -31,7 +31,7 @@ add_custom_command(OUTPUT ${OUTPUT}/timestamp
                    DEPENDS ${DEPS})
 
 add_custom_target(pcapdump ALL DEPENDS ${OUTPUT}/timestamp)
-install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} build -b ${OUTPUT} install --prefix=${BUILD_ROOT}/${CMAKE_INSTALL_PREFIX})")
+install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install --root=${CMAKE_SOURCE_DIR}/debian/tmp --install-layout=deb)")
 
 add_custom_target(btaptap ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/btaptap)
 install(PROGRAMS btaptap DESTINATION ${INSTALL_DEFAULT_BINDIR})
